src/software/simulated_tests/simulated_test_fixture.py

```diff
@@ -60,33 +58,38 @@ def __init__(
 
         """
 
-        self.test_name = test_name
-        self.thunderscope = thunderscope
+        super(SimulatorTestRunner, self).__init__(
+            test_name,
+            thunderscope,
+            blue_full_system_proto_unix_io,
+            yellow_full_system_proto_unix_io,
+            gamecontroller,
+        )
         self.simulator_proto_unix_io = simulator_proto_unix_io
-        self.blue_full_system_proto_unix_io = blue_full_system_proto_unix_io
-        self.yellow_full_system_proto_unix_io = yellow_full_system_proto_unix_io
-        self.gamecontroller = gamecontroller
-        self.last_exception = None
 
-        self.world_buffer = ThreadSafeBuffer(buffer_size=1, protobuf_type=World)
-        self.last_exception = None
+    def set_tactics(
+        self,
+        tactics: AssignedTacticPlayControlParams,
+        team: proto.ssl_gc_common_pb2.Team,
+    ):
```

Comment:
maybe you can pull the common functions into a test_fixture_base.py? (set_tactics, set_play, set_WorldState)